### DISCUSSION TITLE
Symfony 5.4 init after change in servis takes to long

### DISCUSSION THREAD

**Discussion by gBeno** [symfony/symfony#46976] [Q&A]
Hi guys i am currently using Symfony 5.4 and I have a problem in development. I am using auto wiring for my services. As the count of services rises the Symfony takes long to initialise after each change in service it can take up to 60 seconds.  This is no problem for production because it happen only first time but in development when I have to do lot of changes a then test it and again do some other changes and i have to wait for Symfony to reinit after each changes it simply take to long to get something done. Can I somehow tell Symfony to do not reinit after each change in service to speed up the development process. Thx in advance

**Comment by lyrixx** [comment]
Can you provide a reproducer? Thanks
